plotter/__init__.py

```diff
@@ -22,21 +22,22 @@ def main(global_config, **settings):
     config.add_route('upload', 'upload')
     config.add_route('initial', '/initial/{profiles}/{chromosomes}/')
     config.add_route('profile', '/profile/{name}/')
-    config.add_route('old_chrom', '/profile_old/{name}/{chr}/')
+    config.add_route('old_chrom', '/profile/{name}/{chr}/')
+    config.add_route("new_chrom", "/profile_new/{name}/{chr}/")
```

Comment:
I changed the `old_chrom` view to `/profile/` (like it was before GSOC) and I put your `new_chrom` view on `/profile_new/`